runtime/src/test/java/com/getlipa/eventstore/EventStoreTest.java

```diff
@@ -108,6 +112,8 @@ public void testReAppend() throws Throwable {
         final var secondAppended = append(stream, LogIndex.atAny(), second);
         assertEquals(first.getCausationId(), secondAppended.getCausationId());
         assertEquals(firstAppended, secondAppended);
+
+        verify(observer, times(1)).fireAsync(any());
```

Comment:
Why changing it from 3 to 1?

Reply:
I assume you refer to `testAppend`, which appends three different events. However, `testReAppend` is independent and only appends a single event and then re-appends it.